src/test/codegen/issue-75742-format_without_fmt_args.rs

```diff
@@ -0,0 +1,51 @@
+// min-llvm-version: 10.0.0
+// compile-flags: -C opt-level=3
+#![crate_type = "rlib"]
+#![feature(format_args_capture)]
+
+// CHECK-LABEL: @string_from_str
+// CHECK-NOT: __rust_alloc
+// CHECK-NOT: __rust_dealloc
+#[no_mangle]
+pub fn string_from_str() {
```

Comment:
This (and the next one) seem like tests in a wrong context. DCE and malloc optimisation are already tested extensively elsewhere...

Reply:
thanks, I removed this.